### PR TITLE
Inserting Answer Columns in Surveys

### PR DESCRIPTION
**Pull request by KaranSampath**
The source question dataset did not have answers so had to manually solve and insert the column (surveys changed are CRT and ICAR). Would probably need a second look.

### CODE REVIEW

File: surveys/ICAR.csv [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+sister be? ", 35 | 39 | 44 | 47 | 53 | 57,,47
+day,If the day after tomorrow is two days before Thursday then what day is it today? ,Friday | Monday | Wednesday | Saturday | Tuesday | Sunday,,Sunday
+alpha_k,"In the following alphanumeric series, what letter comes next? K N P S U",S | T | U | V | W | X,,X
+alpha_v,"In the following alphanumeric series, what letter comes next? V Q M J H ",E | F | G | H | I | J,,G
+alpha_i,"In the following alphanumeric series, what letter comes next? I J L O S",T | U | V | X | Y | Z,,X
+alpha_q,"In the following alphanumeric series, what letter comes next? Q S N P L",J | H | I | N | M | L,,N
+img_mx1,Please indicate which is the best answer to complete the figure below.,A | B | C | D | E | F,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/MX1.png,E
+img_mx2,Please indicate which is the best answer to complete the figure below.,A | B | C | D | E | F,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/MX2.png,A
```
[the remark this answers is not in the document]
Comment (markwhiting) [reply surveys/ICAR.csv:13]: See above, the order changed from the version in Surveyor re E, so now I think this should be B, instead A.

File: surveys/ICAR.csv [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+img_mx3,Please indicate which is the best answer to complete the figure below.,A | B | C | D | E | F,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/MX3.png,B
+img_mx4,Please indicate which is the best answer to complete the figure below.,A | B | C | D | E | F,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/MX4.png,F
+img_3dr1,All the cubes below have a different image on each side. Select the choice that could represent a rotation of the cube labeled X.,A | B | C | D | E | F | G | H ,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/3DR1.png,C
+img_3dr2,All the cubes below have a different image on each side. Select the choice that could represent a rotation of the cube labeled X.,A | B | C | D | E | F | G | H ,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/3DR2.png,B
+img_3dr3,All the cubes below have a different image on each side. Select the choice that could represent a rotation of the cube labeled X.,A | B | C | D | E | F | G | H ,https://raw.githubusercontent.com/karansampath/rme/master/icar_images/3DR3.png,D
```
Comment (markwhiting) [inline surveys/ICAR.csv:18]: I'd say F. Does that seem reasonable?

File: surveys/ICAR.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (markwhiting) [reply surveys/ICAR.csv:13]: Let me know if that doesn't seem reasonable.

**Review by markwhiting — CHANGES_REQUESTED** [review]

File: surveys/CRT.csv [hunk 3]
```diff
@@ -1,7 +1,7 @@
-name,prompt,response
-drill_hammer,A drill and a hammer cost $330 in total. The drill costs $300 more than the hammer. How much does the hammer cost?,numeric
-rachel,Rachel is the 10th tallest and the 10th shortest girl in her class.  How many girls are in her class?,numeric
-toaster,"When on sale for 20% off, a toaster costs $100.  What does it cost when not on sale?",numeric
-apples,"In a pail of 60 apples, red apples are 3 times more common than green ones. How many are green?",numeric
-eggs,"After hatching from its egg, a baby bird doubles in weight every day. By day 12 it weighs a pound. On what day does the bird weigh half a pound?",numeric
-dog_cat,A dog and a cat weigh 100 pounds in total.  The dog weighs 86 pounds. What is the difference in weight between the dog and the cat?,numeric
+name,prompt,response,answer,group_prompt
+drill_hammer,A drill and a hammer cost $330 in total. The drill costs $300 more than the hammer. How much does the hammer cost?,numeric,30,Please only write the numeric answer to the following questions.
```
[the remark this answers is not in the document]
Comment (KaranSampath) [reply surveys/CRT.csv]: Oh wow that's embarassing

File: surveys/ICAR.csv (hunk 2 above)
Comment (KaranSampath) [reply surveys/ICAR.csv:18]: Yeah was slightly unsure about this but is definitely resonable.

**Comment by KaranSampath** [comment]
Have incorporated comments and rectified RME as well.

**Review by markwhiting — APPROVED** [review]
LGTM!

**Pull request merged**